enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "0.4.1"
+__version__ = "0.5.0"
```

Comment:
@bradenmacdonald I've rebased it to be able to work on #19 and changed the version number to 0.6.0.